lib/WP_Auth0_Options_Generic.php

```diff
@@ -174,16 +169,40 @@ public function set( $key, $value, $should_update = true ) {
 		return $this->update_all();
 	}
 
+	/**
+	 * Remove a setting from the options array.
+	 *
+	 * @param string $key - Option key name to remove.
+	 * @param bool   $should_update - Flag to update DB options array with value stored in memory.
+	 */
+	public function remove( $key, $should_update = true ) {
+		$options = $this->get_options();
+
+		// Cannot remove a setting that is being overridden by a constant.
+		if ( $this->has_constant_val( $key ) ) {
+			return;
+		}
+
+		unset( $options[ $key ] );
+		$this->_opts = $options;
+
+		if ( $should_update ) {
```

Comment:
I don't see any line in this diff passing should_update=true. Since the param is being introduced on this same PR I wonder **why** you need it and if if makes sense to remove it and always (or never) update the db.